### DISCUSSION TITLE
Dreambooth on AUTOMATIC1111

### DISCUSSION THREAD

**Discussion by methodicalculator** [TheLastBen/fast-stable-diffusion#1978] [Q&A]
Hi everyone, I'm quite new to the world so probably this is a dumb question. Why can I not run Dreambooth trough Automatic1111 installed from the Extension? I'd like to use Dreambooth though a different model than stable-diffusion-v-1-5 but I can't a find a way on the Colab project.

Thank you!

**Comment by TheLastBen** [comment]
the dreambooth extension seems not to work with the A1111 notebook, I suggest using the Dreambooth notebook.

**Comment by meunumerotim333** [comment]
I think Dreambooth extension requires A LOT of RAM memory, that's why it doesn't work. Something close to 28Gb of RAM.
